AEPIdentity/Sources/Identity.swift

```diff
@@ -62,6 +62,10 @@ import Foundation
             return state?.readyForSyncIdentifiers(event: event, configurationSharedState: configSharedState) ?? false
         } else if event.type == EventType.configuration, event.source == EventSource.requestIdentity {
             return MobileIdentities().areSharedStatesReady(event: event, sharedStateProvider: getSharedState(extensionName:event:))
+        } else if event.type == EventType.identity, event.source == EventSource.requestIdentity {
+            guard getSharedState(extensionName: IdentityConstants.Hub.SHARED_OWNER_NAME, event: event)?.status ?? .none != .pending else {
```

Comment:
I believe all the dependency checking needs to happen here as returning `false` will let the Event Hub know the extension isn't able to process the event. So you'll need to check if Analytics is registered and check the if the Analytics shared state is pending here.
Also check the event for either `baseUrl` or `urlVariables` so the check only applies to `AppendToUrl` and `GetUrlVariables` requests.